### PR TITLE
Fix the issue #1088 Reparo: Add a support CreateView function 

### LINKED ISSUE #1088: Reparo: Add a support CreateView function
To add a function in Reparo tool when executing binlog data
As in the current version, it does not support "CREATE VIEW Statement".
The reparo main function treats it as invaild ddl type.

### PR DESCRIPTION
**Pull request by lcheung021**
### What problem does this PR solve? 
close #1088
To make reparo tool be able to support "Create View" statement from executing binlog file 

### What is changed and how it works?
Added the Create Statement function in /reparo/ddl.go and /syncer/ddl.go 
 
Tests 
Manual test is performed. (created a new execution file and to restore binlog with "create view" statement.

Related changes
 - Need to re-generate the reparo linux execution program for users to use 

### Release note

- No release note

### CODE REVIEW

File: reparo/ddl.go [hunk 1]
```diff
@@ -77,4 +79,4 @@ func setSchemaIfExists(table *filter.TableName, schemaName string, tableName str
 	if tableName != "" {
 		table.Table = tableName
 	}
-}
+}
```
Comment (dveeden) [inline reparo/ddl.go:82]:
```suggestion
}
```

File: reparo/syncer/translate.go [hunk 2]
```diff
@@ -216,6 +216,8 @@ func parserSchemaTableFromDDL(ddlQuery string) (schema, table string, err error)
 				schema = node.TableToTables[0].NewTable.Schema.O
 			}
 			table = node.TableToTables[0].NewTable.Name.O
+		case * ast.CreateViewStmt:
```
Comment (dveeden) [inline reparo/syncer/translate.go:219]:
```suggestion
		case *ast.CreateViewStmt:
                       if len(node.ViewName.Schema.O) != 0 {
                               schema = node.ViewName.Schema.O
                       }

```

**Review by lichunzhu — APPROVED** [review]
LGTM except for the existing comments

**Review by dveeden — APPROVED** [review]

**Comment by dveeden** [comment]
Please use `go fmt` after changing files to ensure consistent formatting:
```
[dvaneeden@dve-carbon tidb-binlog]$ git rev-parse HEAD
967ef731a751bf5fbc74103559165a3cbdd4fc2c
[dvaneeden@dve-carbon tidb-binlog]$ make check
go version go1.16.8 linux/amd64
gofmt (simplify)
./reparo/ddl.go
make: *** [Makefile:79: fmt] Error 1
[dvaneeden@dve-carbon tidb-binlog]$ go fmt ./reparo/ddl.go
[dvaneeden@dve-carbon tidb-binlog]$ git diff
diff --git a/reparo/ddl.go b/reparo/ddl.go
index 19b8fe1d..5ad9dee5 100644
--- a/reparo/ddl.go
+++ b/reparo/ddl.go
@@ -79,4 +79,4 @@ func setSchemaIfExists(table *filter.TableName, schemaName string, tableName str
        if tableName != "" {
                table.Table = tableName
        }
-} 
\ No newline at end of file
+}
```

**Comment by dveeden** [comment]
LGTM

**Comment by lance6716** [comment]
reparo/reparo.go  100644 → 100755
maybe you have accidentally changed the file permission?

**Review by lance6716 — APPROVED** [review]

**Pull request merged**